### DISCUSSION TITLE
Support plugins by inso

### DISCUSSION THREAD

**Discussion by cseke** [Kong/insomnia#3653] [Ideas]
**Is your feature request related to a problem? Please describe.**
I would like to run my tests automatically, integrated in a CI job (eg: GitLab). Secrets and some other parameters would come from bash environment, however `inso run test` does not support plugins which works well within Insomnia app.

https://github.com/Kong/insomnia/issues/3098#issuecomment-800950722
https://github.com/Kong/insomnia/issues/2593

**Comment by deas** [comment]
Came up with [`deas/insomnia-inso-plugins`](https://github.com/deas/insomnia-inso-plugins) (a hack) to make [`bongosway/insomnia-plugin-dotenv`](https://github.com/bongosway/insomnia-plugin-dotenv) play with `inso`. Guess the approach will work for other plugins as well.
